src/index.js

```diff
@@ -15,6 +15,17 @@ module.exports = async function({
   if (!html) {
     throw Error('You must provide an html property.')
   }
+  let screeshotArgs = {};
+  if (/jpe?g/.test(type)) {
+    if (!quality) {
+      screeshotArgs.quality = 80;
+      quality = 80
+    } else {
+      screeshotArgs.quality = quality;
+    }
+  } else if (type && type !== "png") {
+    throw Error(`Unknow type. Expected 'png','jpg','jpeg' got '${type}'`)
+  }
```

Comment:
We don't need to check if the type is supported. Puppeteer already does it and throws an error.

Reply:
Done by 3f15fd6